modules/ROOT/pages/about-mule-message.adoc

```diff
@@ -150,6 +147,44 @@ Notice that each attribute is a key/value pair (such as `fileName=myJson.json`).
 * `attributes.'fileName'` to return the file name: `myJson.json`.
 * `attributes.size` to return the size of the file: `22`.
 
+=== Valid Identifiers for Attribute Names
+
+Attribute names that follow the rules described in xref:dataweave-language-introduction.adoc#valid-identifiers[Rules for Declaring Valid Identifiers] can be accessed and declared without using any quotation marks.
+
+To use an attribute name that is not a valid identifier, surround the attribute name with any of the following characters:
+
+* single quotation marks (`'`)
+* double quotation marks (`"`)
+* backticks (```)
+
+For example, consider the following DataWeave variable declaration:
+
+[source,json,linenums]
+----
+var myVar = {
+              id : "1234",
```

Comment:
id is showing up inside a filled in red box.

Reply:
yes, this is because the language used to highlight the text. However, this only shows in text editors and in GitHub, but not in the actual build. Just in case, I'll change it.